tests/unit/frameworks/continuous_learning/multihead/multihead_test.py

```diff
@@ -0,0 +1,95 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+Unit tests to ensure that target variables are being transformed properly according to
+the continual learning setup found in the benchmarks repository:
+https://github.com/GMvandeVen/continual-learning
+"""
+
+import unittest
+
+from torch.utils.data import DataLoader
+
+from nupic.research.frameworks.continuous_learning.multihead.subdataset import (
+    get_datasets,
+)
+
+
+def setup(dataset_name, scenario):
+    train_datasets, test_datasets = get_datasets(dataset_name, scenario=scenario)
+    train_loaders = [DataLoader(train_dataset, batch_size=256, shuffle=True)
+                     for train_dataset in train_datasets]
+    test_loaders = [DataLoader(test_dataset, batch_size=1000, shuffle=True)
+                    for test_dataset in test_datasets]
+    return train_loaders, test_loaders
+
+
+class MultiHeadTest(unittest.TestCase):
+
+    def test_task_scenario(self):
+        """
+        Test all target values to be in {0, 1} in the task scenario.
+        """
+        train_loaders, test_loaders = setup("splitMNIST", "task")
```

Comment:
You may need to separate the logic that loads the dataset (`dataset_class`) from the `scenario` functionality to be able to unit test only the `scenario`. Maybe pass `dataset_class` as parameter to `get_datasets`

Reply:
Done.